Convert Python 2.7 to Python 3.6

charlieamartin opened the pull request:
Used the 2to3.py tool to convert the code in Python 2.7 to Python 3.6.  Other changes were made to ensure reading of web pages was successful.

charlieamartin committed:
Convert Python 2.7 to Python 3.6

Used the 2to3.py tool to convert the code in Python 2.7 to Python 3.6.  Other changes were made to ensure reading of web pages was successful.

jimwlewis reviewed (changes requested):
Please keep the filename lines the same as before, so that they just refer to the local directory for now.

I couldn't figure out how to review/accept/reject line-by-line, so it might be all or nothing.

charlieamartin committed:
Changed Filenames to only have filename

This changed the filenames back so that python will only access and create files in the same folder as the scripts.

charlieamartin commented:
I went ahead and updated the filenames so that they just refer to the local directory.

jimwlewis reviewed (approved):
Looks great! Thanks, Charlie.  When you have a chance, please add the two input files: newCalibrationSites.csv and newValidationSites.csv onto GitHub as well.
